simplefi-balancer-v2/package.json

```diff
@@ -4,7 +4,7 @@
   "scripts": {
     "codegen": "graph codegen",
     "build": "graph build",
-    "deploy": "graph deploy --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ nelsongaldeman/simplefi-balancer-v2",
+    "deploy": "graph deploy --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ tarunbhm/simplefi-balance-v2",
```

Comment:
typo here! It should be `balancer`

Reply:
ohh yea, It was for my local debugging graph node. I will push a change with simple-fi account links in package.json. Thanks